testsuite/shared/src/main/java/org/wildfly/test/security/common/elytron/EjbElytronDomainSetup.java

```diff
@@ -132,7 +132,7 @@ public void setup(final ManagementClient managementClient, final String containe
     @Override
     public void tearDown(final ManagementClient managementClient, final String containerId) {
         try {
-            applyUpdate(managementClient.getControllerClient(), Util.getWriteAttributeOperation(remotingConnectorAddress, "sasl-authentication-factory", "application-sasl-authentication"), false);
+            applyUpdate(managementClient.getControllerClient(), Util.getUndefineAttributeOperation(remotingConnectorAddress, "sasl-authentication-factory"), false);
```

Comment:
My guess is this needs to read and cache the value in setup and then either undefine or set it back to the original value here. My guess is the latter is what would be right in many cases where -Delytron is used, hence the failures in the elytron job.

Reply:
fixed